datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCliOptions.java

```diff
@@ -595,6 +596,13 @@ static void ocrLanguage(OptionParser parser) {
                 .ofType(String.class);
     }
 
+    static void ocrType(OptionParser parser) {
+        parser.acceptsAll(List.of(OCR_TYPE_OPT), "OCR implementation: TESSERACT or TESS4J")
+            .withRequiredArg()
+            .ofType(String.class)
+            .defaultsTo("TESSERACT");
```

Comment:
```suggestion
            .defaultsTo(DEFAULT_OCR_TYPE);
```
Could you add the DEFAULT_OCR_TYPE  as a constant at the top of the file this is helpful to maintain the CLI options. Thanks